test_util/marathon.py

```diff
@@ -371,6 +371,6 @@ def deploy_and_cleanup(self, app_definition, timeout=120, check_health=True, ign
         self.destroy_app(app_definition['id'], timeout)
 
     @contextmanager
-    def deploy_pod_and_cleanup(self, pod_definition, timeout=120):
+    def deploy_pod_and_cleanup(self, pod_definition, timeout=180):
```

Comment:
So, this is a value higher than the default value of 120. In that case, a comment here might help too. 
Or we can start with a very high value for the default (300) and stay optimistic that all the tests will be done within that timeframe.